front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -16,20 +16,22 @@ export class AnalysisPage extends Component {
       let analyzedTranscripts = this.props.analyzedTranscripts;
         return (
             <div className="AnalysisPage bg-purple-100 flex h-screen" data-testid="analysis-page">
-              <div className="flex gap-y-10 w-full flex-col h-full">
+              <div className="fixed rounded-b-lg shadow-md inline-grid grid-col-2 justify-between w-full bg-gray-100">
```

Comment:
```suggestion
              <div className="fixed shadow-md inline-grid grid-col-2 justify-between w-full bg-gray-100">
```

I feel very strongly that we should not be rounding the navbar because the edges where it meets the sides look weird.
![image](https://user-images.githubusercontent.com/69657338/202861560-9e7a1b51-a179-4c4f-8b8e-f5f298c7e5cb.png)

Reply:
nice catch on the buttons, I didn't even realize! Also, I'll unround the navbar 👍